src/types.ts

```diff
@@ -383,3 +383,11 @@ export type RequestTimeout = {
   account: number
   full_nodelist: number
 }
+
+// Copied from shardus-core src/shardus/shardus-types.ts
+export declare enum DevSecurityLevel {
+  Unauthorized = 0,
+  Low = 1,
+  Medium = 2,
+  High = 3
+}
```

Comment:
I copied this from `shardus-core` should we bring it in as a dependency instead?